src/styles/sidebar/components/annotation-share-control.scss

```diff
@@ -25,9 +25,15 @@
     bottom: 40px;
   }
 
-  // Override the pointer cursor that applies to the entire annotation card
+  // Override the pointer cursor that applies to the entire card
   cursor: default;
 
+  // Hide the bottom border on the panel's header if displaying
```

Comment:
This is where the variant _with_ the sharing link overrides the generic  panel styling instead of vice-versa.